Content.Server/Body/BodyComponent.cs

```diff
@@ -104,7 +104,7 @@ public override void Gib(bool gibParts = false)
         {
             base.Gib(gibParts);
 
-            SoundSystem.Play(Filter.Pvs(Owner), _gibSound.GetSound(), Owner.Transform.Coordinates, AudioHelpers.WithVariation(0.025f));
```

Comment:
Given this entity is deleted in the next frame, it makes no sense to play this sound on the owner. Revert the change